app/src/main/shared/styles/buttons.scss

```diff
@@ -31,30 +21,26 @@ button:disabled, .button.disabled {
     -webkit-box-shadow: none;
     box-shadow: none;
     opacity: .65;
-    color: #333;
-    background-color: #d4d4d4;
-    border-color: #8c8c8c;
-}
-
-button:disabled:hover, .button.disabled:hover {
-    outline-width: 1px;
+    @include button-variant(#d4d4d4, #333);
```

Comment:
this is going to give it a dark border, just make the border color the same as the background, `#d4d4d4`